manifests/params.pp

```diff
@@ -30,6 +30,10 @@
   $manage_datadir               = true
   $manage_logdir                = true
   $manage_xlogdir               = true
+  $auth_host                    = undef
```

Comment:
instead of adding the variables here, can you please set `undef` as default value in server.pp? that enabled puppet-strings to pickup the values.

Reply:
Sure. That would break the existing style but change needs to starts somewhere